modules/tinymce/src/models/dom/main/ts/table/actions/Clipboard.ts

```diff
@@ -25,15 +25,40 @@ const extractSelected = (cells: SugarElement<HTMLTableCellElement>[]): Optional<
 const serializeElements = (editor: Editor, elements: SugarElement<HTMLElement>[]): string =>
   Arr.map(elements, (elm) => editor.selection.serializer.serialize(elm.dom, {})).join('');
 
-const getTextContent = (elements: SugarElement<HTMLElement>[]): string =>
-  Arr.map(elements, (element) => element.dom.innerText).join('');
+const getTextContent = (editor: Editor, replicaElements: SugarElement<HTMLTableElement>[]): string => {
+  const doc = editor.getDoc();
+  const dos = SugarShadowDom.getRootNode(SugarElement.fromDom(editor.getBody()));
+
+  // Set up offscreen div so that the extracted table element can be inserted into the DOM
+  // TINY-10847: If the table element is detached from the DOM, calling innerText is equivalent to calling
+  // textContent which does not include '\n' and '\t' characters to separate rows and cells respectively
+  const offscreenDiv = SugarElement.fromTag('div', doc);
+  Attribute.set(offscreenDiv, 'data-mce-bogus', 'all');
+  Css.setAll(offscreenDiv, {
```

Comment:
Not sure you need this since you add and remove synchronously there wouldn't be time for the browser do do a paint cycle.

Reply:
That's true. Although, I would prefer to just follow usual offscreen div convention of setting the styles on it to be safe